plugins/net_plugin/net_plugin.cpp

```diff
@@ -1600,10 +1600,10 @@ namespace eosio {
             c->last_handshake_sent.generation = ++c->sent_handshake_count;
             auto last_handshake = c->last_handshake_sent;
             g_conn.unlock();
-            peer_ilog( c, "Sending handshake generation ${g}, lib ${lib}, head ${head}, id ${id}",
+            peer_ilog( c, "Sending handshake generation ${g}, lib ${lib}, fhead ${h}, id ${id}",
                        ("g", last_handshake.generation)
                        ("lib", last_handshake.last_irreversible_block_num)
-                       ("head", last_handshake.head_num)("id", last_handshake.head_id.str().substr(8,16)) );
+                       ("h", last_handshake.head_num)("id", last_handshake.head_id.str().substr(8,16)) );
```

Comment:
should it be: `last_handshake.fork_head_num`

Reply:
Changing the member name of `handshake_message` is likely ok as it is sent in binary form. It would change the FC_REFLECT variant/json format which shouldn't matter, but that is why I didn't originally change it. What do you think?

Reply:
Now the naming is so consistent, that I feel it would be better to update this as well, but up to you if you have any reservations at all.

Reply:
I don't think I do now that I have thought through it. Seems like now is the time to make these types of changes before RC1.